third_party/lighthouse/README.md

```diff
@@ -0,0 +1,31 @@
+Lighthouse
+==========
+
+To setup and run local lighthouse instance, run:
+
+```bash
+npm install
+dart run_suite.dart
+```
+
+Example output:
+
+```
+mobile:                                               avg [per, acc, bps, seo, pwa]
+
+/                                                      72 [ 83,  67,  92,  91,  30]
+/packages                                              71 [ 82,  78, 100,  75,  20]
```

Comment:
May I suggest something like:
```
/packages, avg, 71
  , per, 82
  , acc, 78
  , bps, 100
  , seo, 75
  , pwa, 20
/packages/retry, avg, 73
....
```

So that we can copy/paste into a spreadsheet...